templates/CRM/Activity/Form/Selector.tpl

```diff
@@ -52,12 +54,12 @@
       {/if}
     </td>
 
-  <td>{$row.activity_subject}</td>
+  <td>{if isset($row.activity_subject)}{$row.activity_subject}{/if}</td>
 
     <td>
     {if !$row.source_contact_id}
       <em>n/a</em>
-    {elseif $contactId NEQ $row.source_contact_id}
+    {elseif isset($contactId) && ($contactId NEQ $row.source_contact_id)}
```

Comment:
There is a change here - if `$contactId` was NULL before then the above would be TRUE.  I don't know if that is intentional?

Reply:
It wasn't an intentional change but now I'm trying to figure out what this line is even about. I think it's assuming you're on the activity tab of a contact, and saying if it the activity source is a different contact then make it a link, but (a) why not just make it a link, and (b) this selector is used elsewhere.

Reply:
Ok good catch. This selector seems to only be used on find activities and advanced search displayed as activities, and $contactId is never set. So it's always displaying it as a link before the patch.

Reply:
& the reason not to show a link for logged in user seems a bit opaque too I think?

Reply:
Right. I don't think that's what $contactId is (see also CRM/Activity/Selector/Activity.php, which is a different selector but similar) but if it were the logged in user it would be a strange restriction.